data/src/main/java/org/oppia/android/data/backends/gae/model/GaePlatformParameter.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/** Data class for a Platform Parameter. */
+@JsonClass(generateAdapter = true)
+data class GaePlatformParameter(
+
+  @Json(name = "parameter_name") val parameterName: String,
+  @Json(name = "parameter_value") val parameterValue: Any
```

Comment:
Please add KDocs for these public vals. It may be useful to link to Oppia's backend domain objects to help provide context for these.

Reply:
Done. 
As currently this response structure is not followed in the Oppia backend so we can't have any reference to such objects.